pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -2,27 +2,36 @@ package masteruserrecord
 
 import (
 	"context"
-
+	"fmt"
 	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
-
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
+	"github.com/codeready-toolchain/toolchain-common/pkg/condition"
+	"github.com/go-logr/logr"
+	"github.com/operator-framework/operator-sdk/pkg/predicate"
+	errs "github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/source"
+	"sigs.k8s.io/kubefed/pkg/controller/util"
 )
 
 var log = logf.Log.WithName("controller_masteruserrecord")
 
-/**
-* USER ACTION REQUIRED: This is a scaffold file intended for the user to modify with their own Controller
-* business logic.  Delete these comments after modifying this file.*
- */
+const (
+	// Status condition reasons
+	unableToGetUserAccountReason = "UnableToGetUserAccount"
+	clusterNotReady              = "ClusterNotReady"
```

Comment:
Maybe "TargetClusterNotReady" would be more clear?

Reply:
you are right, renamed in f793b66